### ISSUE TITLE
Better biomes sources

### ISSUE THREAD

**Issue by PssbleTrngle** [PssbleTrngle/Skygrid#67]
The `biome_source` properties for the vanilla dimensions in the world preset json file are set to a simple single-biome source right now. They could probably use better sources, maybe even the same ones used in the regular worldtype [see here](https://github.com/misode/mcmeta/tree/data-json/data/minecraft/dimension)

**Comment by ghost** [comment]
Random thought - is there some way to add TerraBlender compat such that the biome is whatever TB decides it should be?

**Comment by Aceplante** [comment]
Yes please! as of right now it seems to be only plains in the overworld, and only nether wastes in the nether which means no piglins
